helpers/auth.js

```diff
@@ -0,0 +1,45 @@
+function loginRequired(req, res, next) {
```

Comment:
could you please rename this file to authMiddlewares ?

Or maybe move it to middlewares/ directory.